src/SmartComponents/AddSystemModal/SelectedBasket/SelectedBasket.js

```diff
@@ -109,11 +109,12 @@ export class SelectedBasket extends Component {
                 <span>
                     <Popover
                         id='selected-basket'
+                        style={{ minWidth: '500px' }}
```

Comment:
Observation: 500px might be too much for handheld devices. Would it be possible to have it responsive?

Reply:
Yes, and no. I think Patternfly's `Popover` component brings some limitations with it. I tried something like this, but It didn't appear to be responsive at all.
```
.responsive-popover {
  max-width: 500px;
  width: 100%;
}
```
Plus, when there is content in the popover, it doesn't react to it at all, but it does when there is no body content. I even tried adding some CSS to the `bodyContent` and that didn't affect it.

Not to mention, the popover has issues with small screens as it doesn't adjust its orientation based on screen size:
![image](https://user-images.githubusercontent.com/15373136/120819415-8f145880-c521-11eb-833a-35df29b0dd74.png)

If you have suggestions, please let me know. Otherwise, I'm gonna pass on making the responsive for now.

Reply:
All good. :+1:  Responsiveness for such small devices would need more thought, considering also the modal...